core/src/test/java/io/questdb/griffin/WhereClauseParserTest.java

```diff
@@ -1800,7 +1800,7 @@ public void testTwoIntervalsWithOr() throws Exception {
     @Test
     public void testTwoNestedBetween1() {
         try {
-            modelOf("ask between between 1 and 2 and bid+ask/2");
+            modelOf("ask between 1 and 2 and bid+ask/2");
             Assert.fail();
         } catch (SqlException e) {
             TestUtils.assertContains(e.getFlyweightMessage(), "between statements cannot be nested");
```

Comment:
I'm not sure about this. In fact, I suspect it's wrong.

Reply:
two "between" keywords are here on purpose. This is a negative syntax test

Reply:
Dropped. Afaict, this was the failure here:
https://dev.azure.com/questdb/questdb/_build/results?buildId=11535&view=logs&jobId=4955d095-2078-5b5d-31a1-99f71a7c612c&j=4955d095-2078-5b5d-31a1-99f71a7c612c&t=e558e7bf-9ebf-5c2e-7d87-eed2e40883cb